giskard-server/src/main/java/ai/giskard/service/dto/ml/ModelDTO.java

```diff
@@ -1,23 +1,53 @@
 package ai.giskard.service.dto.ml;
 
 import ai.giskard.domain.ml.ProjectModel;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+
 @Getter
 @Setter
 @NoArgsConstructor
 public class ModelDTO {
-    private Long id;
-    private String name;
-    private Long projectId;
-    private String filename;
+    @JsonProperty("id")
+    @NotNull
+    protected Long id;
+    @JsonIgnore
+    protected ProjectDTO project;
+    @JsonProperty("name")
+    protected String fileName;
```

Comment:
we should keep both: `name` and `filename` properties just as in the database table structure

Reply:
check out `ModelSelector.extractModelName` for example that uses both of these properties